### ISSUE TITLE
Security_oM: Add basic definition for electronic access control passage systems

### ISSUE THREAD

**Issue by tiagogrossi** [BHoM/BHoM#1322]
#### Description:
Add a brand new object definition for the security electronic access control passage systems (EACS). Will benefit some Revit workflows.

**Comment by FraserGreenroyd** [comment]
@tiagogrossi has this been prototyped up somewhere first?

**Comment by tiagogrossi** [comment]
@FraserGreenroyd yes got a working prototype for usage in a Revit workflow, should raise a PR soon, just tying up some loose ends elsewhere.
